utils/nctl/sh/staging/set_remotes.sh

```diff
@@ -31,9 +31,13 @@ function _main()
     fi
 
     pushd "$PATH_TO_REMOTES" || exit
-    log "... downloading launcher"
-    curl -O "$_LAUNCHER_URL" > /dev/null 2>&1
+    #log "... downloading launcher"
+    #curl -O "$_LAUNCHER_URL" > /dev/null 2>&1
+    #chmod +x ./casper-node-launcher
+    log "Using launcher from local copy: $NCTL_CASPER_NODE_LAUNCHER_HOME/target/$NCTL_COMPILE_TARGET/casper-node-launcher"
```

Comment:
Are these changes meant to be merged?

Reply:
I think they are a part of Tom's changes to make nctl work with fast sync. So I'm pretty sure they're useful and necessary, but maybe I should also remove the lines that were commented out?

Reply:
Ah, I see: https://github.com/casper-network/casper-node/pull/2577#issuecomment-1012138568
Sorry for the confusion! Not sure what the best approach to merging is; I'm fine with either.

Reply:
the commented lines can be deleted, was using this branch to test the changes and were going to clean these up. Fine with them merging early if they are needed here and I can add the rest of the commits in a later PR.